torch/ao/quantization/quantize_pt2e.py

```diff
@@ -78,6 +98,8 @@ def convert_pt2e(
 
     pm = PassManager([PortNodeMetaForQDQ()])
     model = pm(model).graph_module
+
+    constant_fold(model, _quant_node_filter)
```

Comment:
Do we not want to hide this behind feature flag similar to `user_reference_representation`, because it may break some existing jobs. Or fix those perhaps?

Reply:
will fix the tests

Reply:
this is not just the tests btw. There is xnnpack partitioner and other stuff elsewhere.